backend/frontend/templates/frontend/auth/base.html

```diff
@@ -0,0 +1,22 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en-US" class="h-full bg-gray-100">
+  <head>
+    <meta charset="utf-8" />
+    <title>{% block page_title %}{% endblock page_title %}</title>
+    <script
+      src="https://unpkg.com/htmx.org@1.9.6"
+      integrity="sha384-FhXw7b6AlE/jyjlZH5iHa/tTe9EpJ1Y55RjcgPbjeWMskSxZt1v9qkxLJWNJaGni"
+      crossorigin="anonymous"></script>
+
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+    <script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>
+
+    {% block head %}{% endblock %}
+  </head>
+  <body class="h-full w-full">
```

Comment:
This looks like it's mostly a copy paste of the main template. Is there any way we can make it mre DRY using template inheritance?

Reply:
So why not change the base template and then put the chart stuff in where it is needed? This is a lot of copy-paste